include/OscapScannerBase.h

```diff
@@ -62,8 +62,18 @@ class OscapScannerBase : public Scanner
 
         /// Last read rule id
         QString mLastRuleID;
-        /// If true we are in the rule ID reading phase, if false we are reading rule result
-        bool mReadingRuleID;
+        /// Last downloading file
+        QString mLastDownloadingFile;
+
+        enum Reading_state{
+          READING_PREFIX,
```

Comment:
Prefix all the enum values with "RS_" - acronym for reading state. For example `RS_READING_PREFIX`. Check out `MessageSeverity` for an example.